buildtools/sample-server/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
What do you think about moving the `sample-server` outside of `build-tools` and into the root of this project? That way, the playground is in a more obvious location for developers who are wanting to make changes to `index.html`

Reply:
I actually didn't have an opinion whether it shall live in its own folder and what could be good name (eg. `example`, `sample-app`,  `sample-server`)
Hence keep in buildtools for simplicity for now.

Reply:
moved to `playground`